src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+form {
+  margin-top: 20px;
+  position: relative;
+}
+
+input {
+  width: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: rgba(61, 78, 97, 1);
+  font-family: Avenir, sans-serif;
```

Comment:
always set font to the `html` tag and inherit where needed

Reply:
I moved fonts into HTML selector in css, but code didn't worked, i tried to google it, but could't understand.
Could you please give me one more advice?